my-app/src/components/moves.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+const Moves = ({ history, setXisNext, setStepNumber }) => {
+    const jumpTo = (step) => {
+        setStepNumber(step);
+        setXisNext(step % 2 === 0);
+    };
+
+    const move = history.map((_step, index) => (
```

Comment:
this is semantically wrong because it has many moves in the `history` array. It should be plural not singular. However, why do you need to assign to variable? below implementation
 is totally fine:

```js
<ol>
  {history.map((_step, index) => (
        <li>
             <button onClick={() => jumpTo(index)}>
                 {index ? `Go to move # ${index}` : `Go to game start`}
             </button>
         </li>
  ))}
</ol>
```